osbs/cli/main.py

```diff
@@ -696,6 +697,8 @@ def cli():
                               help='platform name to use')
     worker_group.add_argument('--release', action='store', required=False,
                               help='release value to use')
+    worker_group.add_argument('--arrangement-version', action='store', required=False,
+                              help='default version of inner template to use')
```

Comment:
I'd remove 'default' here. There's no default, which is why it's a required argument.